Common.mk

```diff
@@ -842,7 +842,8 @@ release: $(RELEASE_TARGETS)
 # Iterate over release branch versions, avoiding branches explicitly marked as skipped
 .PHONY: %/release-branches/all
 %/release-branches/all:
-	@for version in $(SUPPORTED_K8S_VERSIONS) ; do \
+	@set -e; \
```

Comment:
We actually have this configured at the top, however the version of make on mac does not support the global flag.  For this specific target, its annoying not to fail on error so adding it specifically.